src/design-system/components/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import DoneIcon from '@mui/icons-material/Done'
+import '~scss-components/tooltip/Tooltip.scss'
+import { cn } from '~/utils/cn'
+
+type TooltipProps = {
+  variant: 'icon' | 'text' | 'icon-text'
+  title: string
+  description?: string
+  position: 'up' | 'down' | 'right' | 'left' | 'none'
+}
+
+const ToolTip: React.FC<TooltipProps> = ({
+  variant,
+  title,
+  description,
+  position
+}) => {
+  const textContent = (
+    <Box className={'s2s-tooltip-text-container'}>
+      <Typography className={'s2s-tooltip-title'}>{title}</Typography>
+      {description && <Typography>{description}</Typography>}
+    </Box>
+  )
+
+  const iconContent = <DoneIcon className={'s2s-tooltip-icon'} />
```

Comment:
Curly braces are unnecessary here, so please remove them.
```suggestion
    <Box className='s2s-tooltip-text-container'>
      <Typography className='s2s-tooltip-title'>{title}</Typography>
      {description && <Typography>{description}</Typography>}
    </Box>
  )

  const iconContent = <DoneIcon className='s2s-tooltip-icon' />
```

Reply:
yeah for sure I didn't notice